src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -222,7 +220,7 @@ public void logAssignment(AssignmentLogData logData) {
             anyString(),
             anyString(),
             any(EppoAttributes.class),
-            anyMapOf(String.class, EppoAttributes.class)
+            any()
```

Comment:
Another change needed for lower version of Mockito. `any()` used here so `nulls` are supported too--needed for the graceful error mode test.